test/Microsoft.AspNet.Razor.Test/Generator/CSharpTagHelperRenderingTest.cs

```diff
@@ -14,10 +14,10 @@ namespace Microsoft.AspNet.Razor.Test.Generator
 {
     public class CSharpTagHelperRenderingTest : TagHelperTestBase
     {
-        private static IEnumerable<TagHelperDescriptor> DefaultPAndInputTagHelperDescriptors
-            => BuildPAndInputTagHelperDescriptors(prefix: string.Empty);
-        private static IEnumerable<TagHelperDescriptor> PrefixedPAndInputTagHelperDescriptors
-            => BuildPAndInputTagHelperDescriptors("THS");
+        private static IEnumerable<TagHelperDescriptor> DefaultPAndInputTagHelperDescriptors { get; }
```

Comment:
prefer the C# 6.0 feature which clearly creates just one `IEnumerable<TagHelperDescriptor>`